packages/internal-test-utils/__tests__/ReactInternalTestUtils-test.js

```diff
@@ -301,3 +307,2079 @@ describe('ReactInternalTestUtils console mocks', () => {
     });
   });
 });
+
+// Helper method to capture assertion failure.
+const expectToWarnAndToThrow = expectBlock => {
```

Comment:
Why the mention of warnings here? Isn't `expectToThrow` sufficient? Same question for `awaitExpectToWarnAndToThrow `

Reply:
Yeah I'll rename, I copied these helpers from the toWarnDev helpers

Reply:
Renamed to `expectToThrowFailure` since it does the ansi stripping for the test failure